.github/workflows/ios_ci.yml

```
@@ -0,0 +1,72 @@
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+          MATCH_KEYCHAIN_NAME: "ci.keychain-db"
+        run: |
+          echo "MATCH_PASSWORD=$MATCH_PASSWORD" > .env.ci
+          echo "KEYCHAIN_PASSWORD=$KEYCHAIN_PASSWORD" >> .env.ci
+          echo "DEV_TEAM_ID=$DEV_TEAM_ID" >> .env.ci
+          echo "DEV_CODE_SIGN_IDENTITY=$DEV_CODE_SIGN_IDENTITY" >> .env.ci
+          echo "APP_IDENTIFIER=$APP_IDENTIFIER" >> .env.ci
+          echo "APPLE_ID=$APPLE_ID" >> .env.ci
+          echo "DISCORD_WEBHOOK_URL=$DISCORD_WEBHOOK_URL" >> .env.ci
+          echo "MATCH_KEYCHAIN_NAME=$MATCH_KEYCHAIN_NAME" >> .env.ci
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)